pom.xml

```diff
@@ -133,11 +133,6 @@
         <artifactId>google-api-services-storage</artifactId>
         <version>v1-rev20191011-1.30.3</version>
       </dependency>
-      <dependency>
-        <groupId>com.google.guava</groupId>
-        <artifactId>guava</artifactId>
```

Comment:
Nice. This is duplicate. Guava is the first entry in this dependencyManagement section.